Cubical/Foundations/Equiv/Dependent.agda

```diff
@@ -222,3 +222,67 @@ equivOver→IsoOver {P = P} {Q = Q} e f equiv = w
   w .inv = isom .inv
   w .rightInv = isom .rightInv
   w .leftInv  = isom .leftInv
+
+
+-- Turn isomorphism over HAE into relative equivalence,
+-- i.e. the inverse of the previous precedure.
+
+isoToEquivOver :
+  {A : Type ℓ } {P : A → Type ℓ'' }
+  {B : Type ℓ'} {Q : B → Type ℓ'''}
+  (f : A → B) (hae : isHAEquiv f)
+  (isom' : IsoOver (isHAEquiv→Iso hae) P Q)
+  → isEquivOver {Q = Q} (isom' .fun)
+isoToEquivOver {A = A} {P} {Q = Q} f hae isom' a = isoToEquiv (fibiso a) .snd
+  where
+  isom = isHAEquiv→Iso hae
+  finv = isom .inv
+
+  fibiso : (a : A) → Iso (P a) (Q (f a))
+  fibiso a .fun = isom' .fun a
+  fibiso a .inv x = transport (λ i → P (isom .leftInv a i)) (isom' .inv (f a) x)
+  fibiso a .leftInv  x = transport
+      (λ i → transport-filler (λ i → P (isom .leftInv a i)) (isom' .inv (f a) (isom' .fun a x)) i
+          ≡ isom' .leftInv a x i) refl
+  fibiso a .rightInv x = transport (λ i → p-path i) (transport (λ i → c-path (~ i)) (isom' .rightInv _ _))
+    where
+    p-path : I → Type _
+    p-path j = PathP (λ i → Q (f (isom .leftInv a (i ∨ j))))
+      (isom' .fun _ (transp (λ i → P (isom .leftInv a (i ∧ j))) (~ j) (isom' .inv (f a) x))) x
+
+    c-path : I → Type _
+    c-path j = PathP (λ i → Q (hae .com a j i)) (isom' .fun _ (isom' .inv (f a) x)) x
+
+
+-- A dependent version of `isoToHAEquiv`
+
+iso→HAEquivOver :
+  {isom : Iso A B}
+  → (isom' : IsoOver isom P Q)
+  → IsoOver (isHAEquiv→Iso (iso→HAEquiv isom .snd)) P Q
```

Comment:
I'm not sure about the name of this function. It's transferring an iso over an iso to an iso over the "improved" equivalence, so maybe `IsoOverIso→IsoOverHAEquiv`?

In some sense it's producing a "dependent HAEquiv", but it's not expressed in the type signature. You'd want some `HAEquivOver` that I guess would require the dependent `com` to lie over the base `com`...

Reply:
Ok, make it. The proof is rather long because Agda's not smart enough... Also I need an extra `JDep` added in `Cubical.Foundations.Prelude`.

Reply:
oh sorry, didn't mean to make you prove all that! I was only asking for the name change. :sweat_smile: But we may as well keep it now that we have it.

Reply:
good news is it's not too hard...